cmd/start.go

```diff
@@ -15,13 +15,16 @@ const (
 	serviceProviderFlagName = "service-provider"
 	socketPathFlagName      = "socket-path"
 	syncProviderFlagName    = "sync-provider"
+	ProviderArgsFlagName    = "sync-provider-args"
 	evaluatorFlagName       = "evaluator"
 	serverCertPathFlagName  = "server-cert-path"
 	serverKeyPathFlagName   = "server-key-path"
 	uriFlagName             = "uri"
 	bearerTokenFlagName     = "bearer-token"
 )
 
+var ProviderArgs map[string]string
```

Comment:
Is this still needed?

Reply:
yes and using as the type in the config now